.github/workflows/build.yaml

```diff
@@ -55,6 +55,22 @@ jobs:
           cd cmake.bld/Linux
           ctest -E mwcsys_executil.t --output-on-failure
 
+  docker_single_node:
+    name: “Check whether docker single-node workflow builds correctly.”
+    runs-on: ubuntu-latest
+    steps:
+      - users: actions/checkout@v3
+      - name: Docker Workflow
+        run: docker compose -f docker/single-node/docker-compose.yaml up --build -d
```

Comment:
We already added a check for a docker image build via single-node compose file, it's in the bottom of the yaml file:

https://github.com/bloomberg/blazingmq/blob/c8acf1b840fcd9e907d4404f43e5e687edd660f5/.github/workflows/build.yaml#L94-L100

Probably we need to unify it, your variant is more advanced so you can just remove the existing one at the bottom.

Reply:
To move this PR, I've done this modification.